week-4/F-object-keys/exercise-part-1.js

```diff
@@ -15,9 +15,11 @@ var mentorsAges = {
 
 // ONLY EDIT BELOW THIS LINE
 
-var mentorsNames = ;
+var mentorsNames = Object.keys(mentorsAges);
 
-var mentorsNamedCapitalised = ;
+var mentorsNamedCapitalised = mentorsNames.map(name =>
+  name.toLocaleUpperCase()
+);
 
```

Comment:
Great - this gives us what we want. You could also use `toUpperCase()`. It might be worth researching the difference between `toUpperCase()` and `toLocaleUpperCase` if you have already 🙂

Reply:
I learned the difference between them thank you very much for reminding me.